src/SFA.DAS.EmployerIncentives.Database/Scripts/Data.CollectionCalendar.sql

```diff
@@ -0,0 +1,26 @@
+﻿/*
+Post-Deployment Script Template							
+--------------------------------------------------------------------------------------
+ This file contains SQL statements that will be appended to the build script.		
+ Use SQLCMD syntax to include a file in the post-deployment script.			
+ Example:      :r .\myfile.sql								
+ Use SQLCMD syntax to reference a variable in the post-deployment script.		
+ Example:      :setvar TableName MyTable							
+               SELECT * FROM [$(TableName)]					
+--------------------------------------------------------------------------------------
+*/
+DELETE FROM incentives.CollectionCalendar
+GO
+INSERT INTO incentives.CollectionCalendar (Id, PeriodNumber, EIScheduledOpenDateUTC, CalendarMonth, CalendarYear) VALUES(12,12,'2021-07-14 08:00:00.000',7,2021)
```

Comment:
This could be simplied to a multi-line insert, i.e.

INSERT INTO incentives.CollectionCalendar (Id, PeriodNumber, EIScheduledOpenDateUTC, CalendarMonth, CalendarYear) VALUES
(12,12,'2021-07-14 08:00:00.000',7,2021),
(11,11,'2021-06-14 08:00:00.000',6,2021)
// etc for rest of calendar entries

Reply:
hmm I didn't to this but if I did it would probably looks very similar (single inserts) , only because that's how SQL management studio generates the data scripts out-the-box. 🤷‍♂️